src/panels/lovelace/editor/config-elements/hui-picture-entity-card-editor.ts

```diff
@@ -0,0 +1,238 @@
+  get _entity(): string {
+    return this._config!.entity || "";
+  }
+
+  get _name(): string {
+    return this._config!.name || "";
+  }
+
+  get _image(): string {
+    return this._config!.image || "";
+  }
+
+  get _camera_image(): string {
+    return this._config!.camera_image || "";
+  }
+
+  get _camera_view(): string {
+    return this._config!.camera_view || "";
+  }
+
+  get _state_image(): {} {
+    return this._config!.state_image || {};
+  }
+
+  get _aspect_ratio(): string {
+    return this._config!.aspect_ratio || "";
+  }
+
+  get _tap_action(): ActionConfig {
+    return this._config!.tap_action || { action: "none" };
+  }
+
+  get _hold_action(): ActionConfig {
+    return this._config!.hold_action || { action: "none" };
+  }
+
+  get _show_name(): boolean {
+    return this._config!.show_name || false;
+  }
+
+  get _show_state(): boolean {
+    return this._config!.show_state || false;
+  }
+
+  protected render(): TemplateResult | void {
+    if (!this.hass) {
+      return html``;
+    }
+
+    const actions = ["more-info", "toggle", "navigate", "call-service", "none"];
+    const views = ["auto", "live"];
+
+    return html`
+      ${configElementStyle}
+      <div class="card-config">
+        <ha-entity-picker
+          .hass="${this.hass}"
+          .value="${this._entity}"
+          .configValue=${"entity"}
+          @change="${this._valueChanged}"
+          allow-custom-entity
+        ></ha-entity-picker>
+        <paper-input
+          label="Name"
+          .value="${this._name}"
+          .configValue="${"name"}"
+          @value-changed="${this._valueChanged}"
+        ></paper-input>
+        <paper-input
+          label="Image Url"
+          .value="${this._image}"
+          .configValue="${"image"}"
+          @value-changed="${this._valueChanged}"
+        ></paper-input>
+        <paper-input
+          label="Camera Image"
+          .value="${this._camera_image}"
+          .configValue="${"camera_image"}"
+          @value-changed="${this._valueChanged}"
+        ></paper-input>
+        <div class="side-by-side">
+          <paper-dropdown-menu
+            label="Camera View"
+            .configValue="${"camera_view"}"
+            @value-changed="${this._valueChanged}"
+          >
+            <paper-listbox
+              slot="dropdown-content"
+              .selected="${views.indexOf(this._camera_view)}"
+            >
+              ${views.map((view) => {
+                return html`
+                  <paper-item>${view}</paper-item>
+                `;
+              })}
+            </paper-listbox>
+          </paper-dropdown-menu>
+          <paper-input
+            label="Aspect Ratio"
+            type="number"
+            .value="${Number(this._aspect_ratio.replace("%", ""))}"
+            .configValue="${"aspect_ratio"}"
+            @value-changed="${this._valueChanged}"
+          ></paper-input>
+        </div>
+        <h3>Toggle Editor to input State Image</h3>
```

Comment:
Does this mean the user has to enter the `yaml` editor to set state image up?

Reply:
correct, I haven't built that out yet. Was going to create a generic state-image editor

Reply:
Not sure if we should show this message, normally we just don't show the function if we don't support it in the editor yet. I don't think this makes it much clearer what to do.

Reply:
I can get behind that